qa-core/src/account-api/api/apis/LicenseAPI.ts

```diff
@@ -20,6 +20,10 @@ export default class LicenseAPI {
         internal: true,
       }
     )
+
+    if (response.status !== 200) {
+      throw new Error(`License Error: ${response.status}`)
```

Comment:
just a small suggestion on more specific logging 
```suggestion
      throw new Error(`Could not update license for accountId=${accountId}: ${response.status}`)
```